pkg/controller/replicaset/replica_set.go

```diff
@@ -547,8 +547,20 @@ func (rsc *ReplicaSetController) syncReplicaSet(key string) error {
 		rsc.manageReplicas(filteredPods, &rs)
 	}
 
+	// Count the number of pods that have labels matching the labels of the
+	// pod template of the replicaSet. Because the label of podTemplateSpec
+	// is a superset of the selector of the replicaset, so the possible matching
+	// pods must be part of the filteredPods.
```

Comment:
Please comment that it's also fine for pods to have more labels than are in the template.